### PR TITLE
 📝 add an overview doc for the keycloak apb

### PR DESCRIPTION
**Pull request by wei-lee**
ping @maleck13 @aidenkeating @matzew

### CODE REVIEW

File: apb_overview.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    * Secrets for database admin users
+    * Service for the Postgres DB
+    * PersistVolume Claim to for the Postgres DB
+    * DeploymentConfig for the Postgres DB
+2. Create a new Keycloak service, including:
+    * DeploymentConfig for Keycloak
+        * Link Keycloak and Postgres DB
+        * Add default Keycloak admin user credential
+    * Service for Keycloak
+    * Routes (both HTTP and HTTPS) for Keycloak
```
Comment (aidenkeating) [inline apb_overview.md]: This should be either HTTP or HTTPS depending on the `keycloak_protocol` variable.

It defaults to a HTTPS route which will redirect any HTTP requests.

**Comment by matzew** [comment]
:eyes:

**Comment by matzew** [comment]
@wei-lee let's add a link to this file, from the `README`, which is the entry point for most users

**Comment by wei-lee** [comment]
@matzew @aidenkeating comments addressed. 

@maleck13 do you think we should create a issue for your comment?

**Pull request merged**